interpretation/profile.py

```diff
@@ -48,6 +49,8 @@ def set_bio(message: Message):
 
 def view_profile(message: Message):
     users = check.users(message, amount=1, delete_duplicates=True, must_be_participant=False)
+    if isParticipant(message.author.id) and not isParticipant(users[0]):
+        return [Mailbox().respond("I am sorry! To prevent any accidental spoilers, you cannot view the profile of dead players.")]
```

Comment:
Nope. Please make this check two lines down when we selected a user. This will break viewing your own profile with `!profile` since then `len(users)` is zero

Reply:
Good call.